content/nb_collectives_intro.tex

```diff
@@ -0,0 +1,28 @@
+An \openshmem nonblocking collective operation, like a blocking collective
+operation, is a group communication operation among the
+participants of the team. All participants of the team are required to call the
+collective operation.
```

Comment:
The "Team-based Collectives" section says:
```
All OpenSHMEM teams-based collective operations are blocking routines. [...]
Team-based collectives must occur in the same program order across all PEs in a team.
```
Do non-blocking collectives also need to occur in the same program order across all PEs in a team?